gradle/libs.versions.toml

```diff
@@ -2,6 +2,7 @@
 kotlin = "1.8.10"
 android_gradle_plugin = "7.4.2"
 koin = "3.4.0"
+ksp = "1.8.10-1.0.9"
```

Comment:
não sabia que funciona isso no .toml, pq precisou ter isso?

Reply:
Eu preciso da versão dele para configurar o plugin: 

`ksp = { id = "com.google.devtools.ksp", version.ref = "ksp"}`
- [Migrate to KSP](https://developer.android.com/build/migrate-to-ksp)